webpack.mix.js

```diff
@@ -46,6 +46,7 @@ mix.scripts([
     './node_modules/cookieconsent/src/cookieconsent.js',
     './node_modules/owl.carousel/dist/owl.carousel.min.js',
     'js/eclipsefdn.videos.js',
+    'js/eclipsefdn.render-rss-feeds.js',
```

Comment:
This would still cause the JS script to crash. We should compress it to a second script, and include that optionally in the ECD tools site instead

Reply:
This is only for testing. We don't use this on production

Reply:
If this output isn't directly consumed by production then +1